src/pages/GalleryPage/GalleryPage.jsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from "react";
+import Modal from "../../components/Modal/Modal";
+import useMetaTags from "../../hooks/useMetaTags";
+import styles from "./GalleryPage.module.css";
+
+const images = Array.from({ length: 13 }, (_, i) => ({
```

Comment:
can't you load all files in folder? without iteration per number, just take all files in folder and return array?

Reply:
It is probably possible to do this using JSON, but I don't know how yet, I need to do some reading and maybe one day I will implement it in this way, but in general the images will be downloaded directly from the google drive, so this implementation is not very important at the moment. The change to downloading from google drive will come in about a month's time.